src/ai/backend/client/func/vfolder.py

```diff
@@ -231,15 +312,21 @@ async def _download_file(
                                     ) as pbar:
                                         loop = current_loop()
                                         writer_fut = loop.run_in_executor(
-                                            None, self._write_file, file_path, file_mode, q.sync_q
+                                            None,
+                                            self._write_file,
+                                            file_path,
+                                            file_mode,
+                                            q.sync_q,
```

Comment:
please make sure that this branch is not behind of main or mistake on work around another branches.
Or is there any reason to change this part?